### PR TITLE
Added additional information to RMA process from customer's account

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Added additional information to RMA process from customer's account

## Affected documentation pages

https://docs.magento.com/user-guide/sales/rma-customer-experience.html

### CODE REVIEW

File: src/sales/rma-customer-experience.md [hunk 1]
```diff
@@ -59,4 +59,9 @@ The Customer completes the following steps on the storefront to submit an RMA:
     ![]({% link images/images/storefront-orders-and-returns-create-new-return.png %}){: .zoom}
     _Create New Return_
 
+1. Additionally, sets **Contact Email Address** and **Comments** if needed.
+
+{:.bs-callout-info}
+If the order contains several items and customer wants to return for example two out of four, they click **Add Item To Return** button, selects an item and sets all mentioned options.
```
Comment (BarnyShergold) [inline src/sales/rma-customer-experience.md:65]:
```suggestion
If the order contains several items and the customer wants to return, for example, two out of four, they click the  **Add Item To Return** button, then select an item and then set all mentioned options.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
A few suggested changes

File: src/sales/rma-customer-experience.md (hunk 1 above)
Comment (DmitryTsymbal) [reply src/sales/rma-customer-experience.md]: Changes applied.
Thanks!

**Review by BarnyShergold — APPROVED** [review]
Thanks!

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**